1.1/spec/11-Part-08.adoc

```diff
@@ -655,14 +660,14 @@ SPARQL language extensions may provide additional associations between operators
 
 This extension mechanism is intended to allow GeoSPARQL implementations to simultaneously support multiple geometry serializations. For example, a system that supports `geo:wktLiteral` serializations may also support `geo:gmlLiteral` serializations and consequently would not raise an error if it encounters multiple geometry datatypes while processing a given query.
 
-Several non-topological query functions use a unit of measure URI. The OGC has defined some standard units of measure URIs under the `+http://www.opengis.net/def/uom/OGC/1.0/+` namespace, for example `+<http://www.opengis.net/def/uom/OGC/1.0/metre>+`.
+Several non-topological query functions use a unit of measure IRI. The OGC has defined some standard units of measure IRIs under the `+http://www.opengis.net/def/uom/OGC/1.0/+` namespace, for example `+<http://www.opengis.net/def/uom/OGC/1.0/metre>+`.
 
 ===== Function: geof:distance
 
 
 ```
 geof:distance (geom1: ogc:geomLiteral, geom2: ogc:geomLiteral, 
-               units: xsd:anyURI): xsd:double
+               units: xsd:anyIRI): xsd:double
```

Comment:
global search-and-replace has bitten here. It is `xsd:anyURI`.

Reply:
Done!